database/seeders/PageTableSeeder.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace EscolaLms\Pages\Database\Seeders;
+
+use EscolaLms\Pages\Models\Page;
+use Illuminate\Database\Seeder;
+use Illuminate\Foundation\Testing\WithFaker;
+
+class PageTableSeeder extends Seeder
+{
+    use WithFaker;
+
+    public function run()
+    {
+        $this->faker = $this->makeFaker();
+
+        Page::factory()
+            ->count(10)
+            ->create()
+        ;
```

Comment:
Imho if the seeder does not perform any more actions, it will needlessly create a new class for this.

Reply:
Fixed

Reply:
Considering what Patryk wrote, we need some sample data for the application. How do you propose to put that in that case then?

Reply:
I think that what @HerbertIV meant was you don't need both `DatabaseSeeder` and `PageTableSeeder` you can either 

* remove `PageTableSeeder` and put this logic in `DatabaseSeeder` or 
* remove `DatabaseSeeder`